util/string_test.cpp

```diff
@@ -10,7 +10,10 @@ using namespace std::literals;
 
 using etest::expect;
 using etest::expect_eq;
+using etest::require;
 using util::no_case_compare;
+using util::split;
```

Comment:
I have nothing against `using namespace util;` to keep this list short. Do you think listing each function individually is better for readability? If we had one test-suite per function I think it'd be clearer, but when we have one larger test-suite for everything in a lib, I'm not so sure.

Reply:
I agree, I'll add `using namespace util;`.